packages/openapi-ts/src/plugins/@tanstack/query-core/plugin.ts

```diff
@@ -1170,6 +1175,9 @@ export const handler: PluginHandler<
                                     {
                                       spread: 'options',
                                     },
+                                    {
+                                      spread: 'localOptions',
```

Comment:
```suggestion
                                      spread: 'mutationOptions',
```

Reply:
Yeah also though about that, I could and it would probably work, but I creates a naming collision with the local variable that is used for typing. A nice solution would be to add support for the `as` keyword of typescript, but I don't want to mess around with this selfmade solution